cinema/migrations/0001_initial.py

```diff
@@ -73,6 +76,9 @@
                 ),
                 ("title", models.CharField(max_length=255)),
                 ("description", models.TextField()),
+                ("duration", models.IntegerField()),
```

Comment:
The `duration` field in the Movie model is an IntegerField, which is suitable for storing the duration in minutes. Ensure that the application logic handles this correctly.